<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kanakadurga" failures="2" tests="2" name="advancedSelenium.SeleniumAdvanced" time="21.667" errors="0" timestamp="2025-10-30T14:10:34 CDT" skipped="0">
  <testcase classname="advancedSelenium.SeleniumAdvanced" name="testScenario" time="12.968">
    <failure message="The following asserts failed:
	expected [TestingWhiz Integration With LambdaTest] but found [Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest],
	expected [https://community.lambdatest.com] but found [https://community.lambdatest.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [TestingWhiz Integration With LambdaTest] but found [Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest],
expected [https://community.lambdatest.com] but found [https://community.lambdatest.com/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at advancedSelenium.SeleniumAdvanced.testScenario(SeleniumAdvanced.java:132)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.run(InvokeMethodRunnable.java:33)
at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:353)
at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:308)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:696)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </failure>
    <system-out>
      <![CDATA[opened windows:[B74F375A88494EF28123F0DFC0C4AF5C, 9F79F2E237DECA662A20550648237869]
Current Window Count:1]]>
    </system-out>
  </testcase> <!-- testScenario -->
  <testcase classname="advancedSelenium.SeleniumAdvanced" name="testScenario" time="8.699">
    <failure message="The following asserts failed:
	expected [TestingWhiz Integration With LambdaTest] but found [Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest],
	expected [https://community.lambdatest.com] but found [https://community.lambdatest.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [TestingWhiz Integration With LambdaTest] but found [Running Automation Tests Using TestingWhiz LambdaTest | LambdaTest],
expected [https://community.lambdatest.com] but found [https://community.lambdatest.com/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at advancedSelenium.SeleniumAdvanced.testScenario(SeleniumAdvanced.java:132)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.run(InvokeMethodRunnable.java:33)
at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:353)
at org.testng.internal.invokers.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:308)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:696)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </failure>
    <system-out>
      <![CDATA[opened windows:[C3E40E9CB1223D5497F76FB37B9F2A13, BC9439B09EB03464AB905643E1148CF5]
Current Window Count:1]]>
    </system-out>
  </testcase> <!-- testScenario -->
  <system-out>
    <![CDATA[opened windows:[B74F375A88494EF28123F0DFC0C4AF5C, 9F79F2E237DECA662A20550648237869]
Current Window Count:1
opened windows:[C3E40E9CB1223D5497F76FB37B9F2A13, BC9439B09EB03464AB905643E1148CF5]
Current Window Count:1]]>
  </system-out>
</testsuite> <!-- advancedSelenium.SeleniumAdvanced -->
